jsk_hand/softhand_ros/udev/80-e151.rules

```diff
@@ -0,0 +1,10 @@
+# Create symlink /dev/{r/l}hand-e151, {r/l}hand-v2-e151
```

Comment:
pleasea add link to product which uses this rules.

Reply:
we need these two to connect dynamixel to usb.
`E151` is the name of the TTL hub board, and the USB dogle is `E160`.
So the correct name should be `r/lhand-e160`...
https://www.besttechnology.co.jp/modules/onlineshop/index.php?fct=photo&p=268
https://www.besttechnology.co.jp/modules/onlineshop/index.php?fct=photo&p=195

Reply:
I update readme and udev files